Core/fft_mt_r2iq.h

```diff
@@ -22,9 +22,8 @@ class fft_mt_r2iq : public r2iqControlClass
     float setFreqOffset(float offset);
 
     void Init(float gain, ringbuffer<int16_t>* buffers, ringbuffer<float>* obuffers);
-    void TurnOn();
-    void TurnOff(void);
-    bool IsOn(void);
+    void TurnOn() override;
+    void TurnOff(void) override;
```

Comment:
please keep consistent. I prefer not put void here.

Reply:
OK.